### PR TITLE
notify-136 change financial year starting in april to calendar year

### PR DESCRIPTION
**Pull request by terrazoon**

### CODE REVIEW

File: app/dao/date_util.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def get_months_for_year(start, end, year):
 
 
 def get_financial_year(year):
```
Comment (stvnrlly) [inline app/dao/date_util.py:25]: Let's update this function name to some thing like `get_calendar_year()`

File: app/dao/date_util.py [hunk 2]
```diff
@@ -37,21 +37,20 @@ def get_financial_year_dates(year):
 
 def get_current_financial_year():
```
Comment (stvnrlly) [inline app/dao/date_util.py:38]: As above, let's update the function name.

File: app/dao/date_util.py [hunk 3]
```diff
@@ -37,21 +37,20 @@ def get_financial_year_dates(year):
...
     current_year = int(now.strftime('%Y'))
-    year = current_year if current_month > 3 else current_year - 1
+    year = current_year
     return get_financial_year(year)
 
 
-def get_april_fools(year):
+def get_new_years(year):
     """
      This function converts the start of the financial year April 1, 00:00 as BST (British Standard Time) to UTC,
```
Comment (stvnrlly) [inline app/dao/date_util.py:47]: This comment should be updated to reflect the change.

**Review by stvnrlly — CHANGES_REQUESTED** [review]

**Review by ccostino — COMMENTED** [review]
I took another look through this with the changes made from the previous suggestions and it looks good to go from my side!

@stvnrlly, was there anything else that had to happen with this before we approve?

**Comment by ccostino** [comment]
Noting that given the age of the PR we may need to update with `main`, but it looks like this can still be merged without an issue according to GitHub.

**Review by stvnrlly — APPROVED** [review]

**Pull request merged**